### PR TITLE
documentation about microservice intra-namespace communication

### PR DESCRIPTION
**Pull request by elmalakomar**
## Pull Request Type

- [ X] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [ ] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [ ] No sensible content has been committed

### CODE REVIEW

File: docs/how_to/console/project_development.mdx [hunk 1]
```diff
@@ -1,10 +1,31 @@
 ---
-id: project_dev_tools_setup
-title: Project Development Tools Setup
-sidebar_label: Project Development Tools Setup
+id: project_development
```
Comment (fredmaggiowski) [inline docs/how_to/console/project_development.mdx:2]: I think this change produces a different URL, doesn't it? If that's the case we have to map the old url in the 301redirects.json file.

File: docs/how_to/console/project_development.mdx [hunk 2]
```diff
@@ -1,10 +1,31 @@
...
+
+## Microservice communication intra-namespace
+
+In this guide you will see how microservices can interact and communicate inside the same namespace.
+
+The Mia-Platform Console offers an overview for all microservices created in which is visible a section [Container ports](/development_suite/api-console/api-design/microservice-container-ports.md) where you can view and manage container ports.
+
+![Environment variable section](img/container_ports.png)
+
+Suppose you have two microservices in the same namespace; `service-A` that needs to call `service-B` on the `GET /my-api` endpoint.
```
Comment (fredmaggiowski) [inline docs/how_to/console/project_development.mdx:19]: service names in k8s do not accept upper case characters; i'd not make an example with `service-A` but with `service-a`

**Review by fredmaggiowski — CHANGES_REQUESTED** [review]

File: docs/how_to/console/project_development.mdx [hunk 3]
```diff
@@ -1,10 +1,31 @@
...
+To do so `service-A` has to make a request equivalent to the following cURL:
+
+```curl
+curl --request GET 'http://service-B:80/my-api'
```
[the remark this answers is not in the document]
Comment (peppemanzi) [reply docs/how_to/console/project_development.mdx:23]: Maybe we can use generic HTTP request:

```
GET /my-api HTTP/1.1
Host: service-b:80
```

File: docs/how_to/console/project_development.mdx [hunk 4]
```diff
@@ -1,10 +1,31 @@
...
-In this guide you will see how to configure a list of development tools in order to use them in the Mia-Platform Console.
+In this guide you will see how to configure:
+- services inside the Mia-Platform Console
+- development tools in order to use them in the Mia-Platform Console.
+
+## Microservice communication intra-namespace
+
+In this guide you will see how microservices can interact and communicate inside the same namespace.
+
+The Mia-Platform Console offers an overview for all microservices created in which is visible a section [Container ports](/development_suite/api-console/api-design/microservice-container-ports.md) where you can view and manage container ports.
```
[the remark this answers is not in the document]
Comment (peppemanzi) [reply docs/how_to/console/project_development.mdx:15]: IMHO this sentence is not so clear. I would make more explicit what are the steps to undestand how to call a service from another one before making the example.

I would suggest something like:
```md
If you want to call a microservice from within its namespace, you need to use:

-  as hostname, the target microservice's name;
-  as port, the "Port" of the container exposing the endpoint you want to call.

The value set for such port can be find the in the "Port" column of [Container ports](/development_suite/api-console/api-design/microservice-container-ports.md) section of the micorservice detail .
```

**Review by peppemanzi — CHANGES_REQUESTED** [review]

File: docs/how_to/console/project_development.mdx (hunk 4 above)
[the remark this answers is not in the document]
Comment (elmalakomar) [reply docs/how_to/console/project_development.mdx]: changed

File: docs/how_to/console/project_development.mdx (hunk 1 above)
Comment (elmalakomar) [reply docs/how_to/console/project_development.mdx:2]: changed

File: docs/how_to/console/project_development.mdx (hunk 2 above)
Comment (elmalakomar) [reply docs/how_to/console/project_development.mdx]: changed

**Review by peppemanzi — APPROVED** [review]

**Review by peppemanzi — APPROVED** [review]

**Review by fredmaggiowski — APPROVED** [review]

**Pull request merged**